common/shlibs

```diff
@@ -814,6 +814,10 @@ libpeas-gtk-1.0.so.0 libpeas-1.0.0_2
 libtotem.so.0 libtotem-3.6.2_1
 libclutter-1.0.so.0 clutter-1.14.0_1
 libavfs.so.0 libavfs-1.0.5_1
+libclutter-1.0.so.0 deepin-clutter-1.26.2_1
+libdeepin-mutter.so.0 deepin-mutter-3.20.29_1
+libanything.so.1.0.0 deepin-anything-0.0.1_1
+libdmr.so.0.1 deepin-movie-reborn-3.2.5_1
```

Comment:
is anything using deepin-movie-reborn libs ?

Reply:
Not yet, but soon it will. ( https://github.com/Johnnynator/void-packages-1/tree/deepin )
I also just repushed, to remove the wrong entries in `common/shlibs` that git rebase put in there.